projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3191,7 +3191,7 @@ public function throw_error_on_activate_plugin( $plugin ) {
 
 			if ( $throw ) {
 				/* translators: Plugin name to deactivate. */
-				throw new RuntimeException( sprintf( __( 'Jetpack contains the most recent version of the old “%1$s” plugin.', 'jetpack' ), 'WordPress.com Stats' ) );
+				throw new RuntimeException( sprintf( __( 'Jetpack contains the most recent version of the old "%1$s" plugin.', 'jetpack' ), 'WordPress.com Stats' ) );
```

Comment:
This is just cleanup leftovers from an earlier commit (never relevant, but I couldn't really change it back to the wrong quote format). So, just needs treating like a linting change.